docs/test/misc/ds4/alignment-controls-grid.html

```diff
@@ -27,7 +27,7 @@ <h2>Small</h2>
                     <option value="3">Option 3</option>
                 </select>
                 <span class="select__icon" hidden>
-                    <svg aria-hidden="true" focusable="false">
+                    <svg hidden="true" focusable="false">
```

Comment:
Is should just be `hidden` with no assignment of value.

Reply:
Fixed this by removing the unneeded svg completely